README.md

```diff
@@ -230,6 +230,19 @@ This would restrict substring searches to the ```username``` attribute of the Us
 $ curl http://localhost/users?searchOnlyUsernames=james
 ```
 
+By default, the substring search is performed using a ```{field} LIKE '%{query}%'``` pattern. However, this behavoir can be customized by specifying a search operator. Valid operators include: `$ne`, `$not`, `$gte`, `$gt`, `$lte`, `$lt`, `$like` (default), `$ilike`/`$iLike`, `$notLike`, `$notILike`. For instance:
```

Comment:
That would be my preference, but we can only support operators that take in a string. This rules out $in, $contains, etc.

Definitely open to other approaches on this.